terraform/tls-web-application/terraform/main.tf

```diff
@@ -0,0 +1,106 @@
+terraform {
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
+      version = "3.0.2"
+    }
+    dnsimple = {
+      source  = "dnsimple/dnsimple"
+      version = "~> 1.1.2"
+    }
+  }
+}
+
+provider "docker" {}
+
+provider "dnsimple" {
+  token   = var.dnsimple_token
+  account = var.dnsimple_account
+}
+
+variable "dnsimple_token" {
+  description = "DNSimple API Token"
+  type        = string
+  sensitive   = true
+}
+
+variable "dnsimple_account" {
+  description = "DNSimple Account ID"
+  type        = string
+}
+
+variable "dnsimple_domain" {
+  description = "DNSimple Domain"
+  type        = string
+}
+
+resource "dnsimple_registered_domain" "domain" {
+  name = "${var.dnsimple_domain}"
+
+  contact_id            = 45600
+  auto_renew_enabled    = true
+  whois_privacy_enabled = true
+  dnssec_enabled        = false
+}
```

Comment:
Have you considered omitting this resource from the config as it is not required, or maybe using the `dnsimple_domain` resource instead?

Reply:
Updated to use `dnsimple_domain`